src/Rings/mpoly-ideals.jl

```diff
@@ -1325,3 +1326,7 @@ function minimal_generating_set(I::MPolyIdeal{<:MPolyElem_dec}; ordering::Monomi
 
   return elem_type(R)[ R(f) for f in gensS ]
 end
+
+function grassman_pluecker_ideal(subspaces_dimension::Int, amb_dimension::Int)
+    return Polymake.ideal.pluecker_ideal(subspaces_dimension, amb_dimension)
```

Comment:
What type of object does this return? Can we get a test case (eg as part of a docstring example)?